src/client/sandbox/shadow-ui.js

```diff
@@ -56,8 +57,15 @@ export default class ShadowUI extends SandboxBase {
         return filteredList.length === nlLength ? list : filteredList;
     }
 
-    static _filterNodeList (nodeList) {
-        return ShadowUI._filterList(nodeList, item => ShadowUI._filterElement(item));
+    static _filterNodeList ({ nodeList, isLiveCollection }) {
```

Comment:
I don't see so big problem here, but I vote for `ShadowUI._filterNodeList(nodeList, isLiveCollection)` to be consistent with other filter methods (filterElement, filterList) where the first argument is an element/list.